test/e2e/network_test.go

```diff
@@ -26,6 +26,7 @@ func testNetwork(t *testing.T) {
 	defer testCtx.deleteNamespace(testCtx.workloadNamespace)
 	t.Run("East West Networking across Linux and Windows nodes", testEastWestNetworking)
 	t.Run("East West Networking across Windows nodes", testEastWestNetworkingAcrossWindowsNodes)
+	t.Run("Metrics test", testMetrics)
```

Comment:
This test need not be a part of testNetwork(), it can be added to the list of tests in create_test.go creationTestSuite